shallow_backup/__main__.py

```diff
@@ -31,7 +32,7 @@
 @click.option('-separate_dotfiles_repo', is_flag=True, default=False, help="Use if you are trying to maintain a separate dotfiles repo and running into issue #229.")
 @click.option('-show', is_flag=True, default=False, help="Display config file.")
 @click.option('--version', '-v', is_flag=True, default=False, help='Display version and author info.')
-def cli(all, configs, delete_config, destroy_backup, dotfiles, fonts, new_path,
+def cli(add, all, configs, delete_config, destroy_backup, dotfiles, fonts, new_path,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'all'](https://app.codacy.com/manual/alichtman/shallow-backup/pullRequest?prid=4957117)